CHANGELOG.md

```diff
@@ -28,6 +32,7 @@
 
 * Add lazy loading to image card ([PR #2270](https://github.com/alphagov/govuk_publishing_components/pull/2270))
 * GOVUK.Modules.start can accept DOM elements ([PR #2260](https://github.com/alphagov/govuk_publishing_components/pull/2260))
+* Create devolved nations component ([PR #2280](https://github.com/alphagov/govuk_publishing_components/pull/2280))
```

Comment:
Looks like the changelog entries have been confused a little - this should only be under `Unreleased`.

Reply:
Ah sorry missed that one when I rebased. Now fixed.